code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -7,12 +7,14 @@
 	species = "replicapod"
 	plantname = "Replica Pod"
 	product = /mob/living/carbon/human //verrry special -- Urist
+	container_type = INJECTABLE|DRAWABLE
```

Comment:
I did it this way so that you can replace the sample in case you put in the wrong one. Seeds don't really have any interactions with anything, so I didn't think this would be an issue.